cl/tests/test_feeds.py

```diff
@@ -0,0 +1,154 @@
+# coding=utf-8
+"""
+Functional testing of courtlistener RSS feeds
+"""
+from django.conf import settings
+from django.test.utils import override_settings
+from cl.search.models import Court
+from cl.tests.base import BaseSeleniumTest
+import feedparser
+import os
+
+
+@override_settings(
+    MEDIA_ROOT = os.path.join(settings.INSTALL_ROOT, 'cl/assets/media/test/')
+)
+class FeedsFunctionalTest(BaseSeleniumTest):
+    """Tests the Feeds page and functionality"""
+
+    fixtures = ['test_court.json', 'judge_judy.json', 'functest_opinions.json',
+        'functest_audio.json']
+
+    def test_can_get_to_feeds_from_homepage(self):
+        """Can we get to the feeds/podcasts page from the homepage?"""
+        self.browser.get(self.server_url)
+        link = self.browser.find_element_by_link_text('Feeds & Podcasts')
+        link.click()
+
+        self.assertIn('Feeds & Podcasts', self.browser.title)
+        self.assertIn('/feeds', self.browser.current_url)
+        self.assert_text_in_body('Feeds & Podcasts')
+
+    def test_feeds_page_shows_jurisdiction_links(self):
+        """
+        Does the feeds page show all the proper links for each jurisdiction?
+        """
+        courts = Court.objects.all()
+        self.browser.get('%s/feeds' % (self.server_url,))
+        self.assert_text_in_body('Jurisdiction Feeds for Opinions')
+
+        for court in courts:
+            link = self.browser.find_element_by_link_text(court.full_name)
+            self.assertEqual(
+                link.get_attribute('href'),
+                '%s/feed/court/%s/' % (self.server_url, court.pk,)
```

Comment:
Ditto here. This one needs a name added to it though in `urls.py` (it lacks a name presently, alas), but then can be called with something like:

    reverse('jurisdiction-feed', kwargs={'court': court.pk})